ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,205 @@
+
+  - Active (blue)
+  - Done (green)
+  - Error (red)
+  - Error; automatic retry (red?)
+  - Suspended (grey)
+  - Process aborted (black)
+
+ [comment]: <> (Das sind die Status der Actions Drop-down-Liste. Wenn ein Action wartet, ist die Farbe in der Graphik weiß. Hinzufügen? Farben stimmen oder ändern sich noch?)
+
+4. Click the action to be checked in the diagram.  
+The selected action is displayed in the *Actions* tab.
+
+5. Check the action status in the *Status* column. The following statuses can be displayed:  
+
+    - *Active*  
+    - *Done*  
+    - *Error*  
+    - *Error; automatic retry*  
+    - *Suspended*  
+    - *Process aborted*
+
+
+6.  Check the number of tries for the action in the *Tries* column.
+
+7. Check the number of times an action must be retried after error in the *Maximal retries after error* column.
+
+8. Check the assigned queue type in the *Queue type* column.
+
+  > [Info] The queue type is assigned to an action when editing a workflow, see [Edit an action](01_ManageWorkflows.md#edit-an-action).
+
+9.  Check the time when the action is being tried in the *Defer until* column.
+
+> [Info] A list of all actions, regardless of the process they are part of, can be displayed in the *Process Actions* tab. There they can be filtered according to their status and modified simultaneously, if necessary. By clicking a specific action in the process actions list, the corresponding *Process ID* view will be displayed. For detailed information, see [Process actions](ActindoWorkFlow/UserInterface/04a_ProcessActions.md).
+
+[comment]: <> (Bezug zu Logs hier? Wenn man ein Action auswählt und dann auf Logs tab klickt, zeigt Logs tab alle Logs, nicht das Log, das sich auf die ausgewählte Action bezieht. Bug oder erwartetes Verhalten?)
+
+### Next steps
+
+- [Check the token status and content](#check-the-token-status-and-content)
+
+### See also
+
+- [Retry a single process action](/ActindoWorkFlow/Troubleshooting/02_RetryProcessAction.md#retry-a-single-process-action)
+- [Retry multiple process actions](/ActindoWorkFlow/Troubleshooting/02_RetryProcessAction.md#retry-multiple-process-actions)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Workflows](/ActindoWorkFlow/UserInterface/02a_Workflows.md)
+- [Processes](/ActindoWorkFlow/UserInterface/03a_Processes.md)
+- [Process actions](/ActindoWorkFlow/UserInterface/04a_ProcessActions.md)
+
+
+## Check the token status and content
+
+The token status and content can be checked to monitor how data progress within a process and, if necessary, detect and solve any data type related issues.   
+
+The *Tokens* tab displays the tokens being processed. A token is a container carrying any data needed to complete an action, which could be a number or a document. When a operation is performed, a data entity is generated, for instance an invoice, which in turn initiates the workflow process.
```

Comment:
```suggestion
The *Tokens* tab displays the tokens being processed. A token is a container carrying any data needed to complete an action, which could be a number or a document for example.
```
I don't understand what the last sentence is supposed to tell me, not relevant when I'm currently looking at a process and its tokens